src/Nox.Types/Types/Area/Area.cs

```diff
@@ -6,7 +6,7 @@ namespace Nox.Types;
 /// <summary>
 /// Represents a Nox <see cref="Area"/> type and value object.
 /// </summary>
-public class Area : Measurement<Area, AreaUnit>
+public sealed class Area : Measurement<Area, AreaUnit>
```

Comment:
ok.

We already saw that probably we will need to change this because o EF LayzLoadProxy requirements

Reply:
Thanks for the info. Removed in all measurement types.